.tools/validate.go

```diff
@@ -0,0 +1,269 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+)
+
+// DefaultRules are the standard validation to perform on git commits
+var DefaultRules = []ValidateRule{
+	func(c CommitEntry) (vr ValidateResult, err error) {
+		vr.CommitEntry = c
+		hasValid := false
+		for _, line := range strings.Split(c.Body, "\n") {
+			if validDCO.MatchString(line) {
+				hasValid = true
+			}
+		}
+		if !hasValid {
```

Comment:
`<nit>`To avoid a double-negative (`!hasValid {} else {}`), maybe drop the `!` and flip the if/else clauses?`</nit>`